checker/build.xml

```diff
@@ -550,6 +550,8 @@
               </fileset>
           </batchtest>
       </junit>
+
+      <antcall target="units_extension-tests"/>
```

Comment:
Don't use underscores in the new targets (also below). All other targets use dash as separator.